src/main/java/duke/models/Storage.java

```diff
@@ -0,0 +1,101 @@
+package duke.models;
+
+import duke.exceptions.DukeException;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.Scanner;
+
+/**
+ * The storage class provides methods to load and save tasks to disk.
+ *
+ * @author Zhu Yuanxi
+ */
+public class Storage {
+    private static final String TASK_DIRECTORY_NAME = "src/main/data/";
```

Comment:
Perhaps file address should be passed into the constructor so that Storage can be used for other files as well